### DISCUSSION TITLE
How to create a parameter where a user can manually enter 1 or more ID's (not select from a list)

### DISCUSSION THREAD

**Discussion by stevehydro** [eclipse-birt/birt#1499] [Q&A]
Hi,

I use Version: Neon.3 Release (4.6.3) Build id: 20170314-1500 and need to create a parameter where the user can type 1 or more IDs.

I cannot find out how to do this, is it possible?

Attached is an example from Crystal Reports that I want to replicate.
![Capture](https://github.com/eclipse-birt/birt/assets/142328454/bb3c08ab-2b8d-4091-ae4d-988e16f3a60f)

Thank-you!

**Comment by speckyspooky** [comment]
This option is currently not given on BIRT-parameters.
This kind of parameter would be a complietely new parameter type which must be implementen (= feature request).

**Comment by hvbtup** [comment]
Furthermore, such parameters would need a completely different handling in data sets, eg. in JDBC data sets.
